docs/essentials/controls.md

```diff
@@ -84,14 +84,14 @@ This replaces the input with a radio group for a more intuitive experience.
 
 ## Custom control type matchers
 
-For a few types, Controls will automatically infer them by using [regex](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp). You can change the matchers for a regex that suits you better.
+For a few types, Controls can automatically be infered with [regex](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp). If you've used the Storybook CLI to setup the configuration it will automatically have created the following defaults in `.storybook/preview.js`:
 
```

Comment:
I see what you mean about making it simpler, but I don't think it's grammatically correct. (I know I'm nit picking here, but it's actually just to get better at English writing).

First part of the sentence:

> If you've used the Storybook CLI to setup the configuration...

is [present perfect tense](https://learnenglish.britishcouncil.org/grammar/english-grammar-reference/present-tense). the user has **at some point in time previously** used the CLI to set up the config.

Second part:

> ... it will automatically create the following defaults in `.storybook/preview.js`:

is present simple tense. The CLI will **now at this moment** create the defaults.

Which it won't. The defaults have already been created, they won't be created now.


I could be totally wrong though, English is not my native language.

Reply:
@jonniebigodes should we get this done?

Varun suggested this alternative, which I like:
> For a few types, Controls can automatically be inferred with [regex](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp). If you've used the Storybook CLI to setup your project it should have automatically created the following defaults in `.storybook/preview.js`:

I'm also fine with going with your suggestion, so we can get it over the finish line 🙂, you have the final say.

Reply:
@JReinhold, that works for me! I'm good with Varun's suggestion. Feel free to update it accordingly, and let's get this one in.